src/MealCard/MealCard.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import moment from "moment";
+import "./MealCard.less";
+
+function formatPrepTimeHours(prepTime) {
+	let hours = Math.floor(parseInt(prepTime) / 60);
+
+	if(isNaN(hours)) {
+		return "";
+	}
+	else if(hours === 0) {
+		return "";
+	} else if(hours === 1) {
+		return `${hours} hour`;
+	} else {
+		return `${hours} hours`;
+	}
+}
+
+function formatPrepTimeMinutes(prepTime) {
+	let minutes = parseInt(prepTime) % 60;
+
+	if(isNaN(minutes)) {
+		return "";
+	} else if(minutes === 0) {
+		return "";
+	} else if(minutes === 1) {
+		return `${minutes} minute`;
+	} else {
+		return `${minutes} minutes`;
+	}
+}
+
+function formatPrepTime(prepTime) {
+	return `${formatPrepTimeHours(prepTime)} ${formatPrepTimeMinutes(prepTime)}`;
+}
+
+function formatLastUsed(lastUsed) {
+	const date = moment(lastUsed);
+
+	if(lastUsed && date.isValid()) {
+		return date.format("MMM Do, YYYY");
+	} else {
+		return "never used";
+	}
+}
```

Comment:
these are equivalent to the computed properties in the vue version of this component, but without being tied to the "computed" object, and without using shared state. The fact that this version is decoupled is awesome, but the extra organization imposed by the vue structure is rather nice.